### ISSUE TITLE
Wrong height on Landing Page

### ISSUE THREAD

**Issue by synoet** [BUGS-NYU/cs-resources#21]
![Screen Shot 2021-03-19 at 7 08 31 PM](https://user-images.githubusercontent.com/10552019/111850374-7fa04e80-88e6-11eb-8fe0-673fd8e1da8e.png)

**Comment by Kn99HN** [comment]
Yeah this is because of the triangle gradient bleeding out onto the page

**Comment by esilverm** [comment]
This seems to not be happening on my browser, can you provide more details about which browser you are experiencing this problem on?

**Comment by Kn99HN** [comment]
I think this only happens with bigger resolution like 2560x1440. That's how I inspect what happened

**Comment by esilverm** [comment]
I dont have any screens that large so I may need somebody else to take care of this bug
